bazel/repository_locations.bzl

```diff
@@ -1385,9 +1385,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Intel Dlb",
         project_desc = "Dlb",
         project_url = "https://networkbuilders.intel.com/solutionslibrary/queue-management-and-load-balancing-on-intel-architecture",
-        version = "7.6.0",
-        sha256 = "b74c1bb2863fb6374bf80b9268b5978ab7b9d4eabb2d47ea427a5460aa3ae5fe",
-        urls = ["https://downloadmirror.intel.com/727424/dlb_linux_src_release7.6.0_2022_03_30.txz"],
+        version = "7.8.0",
+        sha256 = "9070388d9a22055ca37c7e1ec6a4b55c6bfd829b1a99b860056af268b68c98b6",
+        urls = ["https://downloadmirror.intel.com/744007/dlb_linux_src_release_7.8.0_2022_09_20.txz"],
```

Comment:
This line should be changed to not hard code the version`        urls = ["https://downloadmirror.intel.com/744007/dlb_linux_src_release_{version}_2022_09_20.txz"],`. Also should the year in the version string `2022_09_20` match the `release_date` value for this dependency which remained unchanged at `2022-03-30`?

This is a good example why Github hosted dependencies are preferred so the meta data can be validated.

Reply:
Thanks, I will pay attention the next time.